### ISSUE TITLE
Calling method via UA_Client_call fails with UA_STATUSCODE_BADINVALIDARGUMENT

### ISSUE THREAD

**Issue by keba-estr** [open62541/open62541#5554]
A Method is transferred to the server with the abstract type "structure" (ns=2, id=22) for its arguments. The server does not allow the abstract type "structure" in the function checkAdjustArguments (ua_services_method.c) and rejects the call with UA_STATUSCODE_BADINVALIDARGUMENT.

Problem found in master branch (commit version 42bd8e1c98c230ffb9d99c6c6c426ce2da175514)

Occurs with method SendCyclicJobList of Euromap83.

Worked fine with version 1.3.3.

## Background Information / Reproduction Steps

Used CMake options:

```bash
cmake -DUA_NAMESPACE_ZERO=<YOUR_OPTION> <ANY_OTHER_OPTIONS> ..
```

## Checklist

Please provide the following information:

 - [X] open62541 Version (release number or git tag): 42bd8e1c98c230ffb9d99c6c6c426ce2da175514
 - [ ] Other OPC UA SDKs used (client or server): 
 - [ ] Operating system:
 - [ ] Logs (with `UA_LOGLEVEL` set as low as necessary) attached
 - [ ] Wireshark network dump attached
 - [ ] Self-contained code example attached
 - [ ] Critical issue

**Comment by keba-estr** [comment]
Seems to be resolved with PR #5360.

**Comment by keba-estr** [comment]
Already resolved with PR #5360.
Extension object is decoded before checking type.
